### ISSUE TITLE
Response to refresh message doesn't send window definitions

### ISSUE THREAD

**Issue by curiousdannii** [erkyrath/remglk#34]
The response to a refresh message doesn't send window definitions.

Also, would it be possible for refresh messages to not need to send the generation?

**Comment by erkyrath** [comment]
Yeah, having the generation default to zero (for "refresh") is reasonable.
